src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -341,24 +341,39 @@ class SftpServer
 
         attr.size = file_info.size();
 
-        auto map = uid_map.find(file_info.ownerId());
-        if (map != uid_map.end())
+        const auto no_id_info_available = static_cast<uint>(-2);
```

Comment:
Would making this a constexpr declared in the anonymous namespace be more efficient?

Reply:
Using https://godbolt.org/, it doesn't seem like it makes a difference.

Reply:
Doing an assignment each time we enter that function vs. setting the no_id_info_available at compile is the same efficiency?  It just seemed like a candidate for global definition that is set one time.

Reply:
Yeah, looking at the assembly output, even without optimization, the compiler just uses a constant value, no storage allocated for it.

Reply:
Oh actually, I take that back. It's under -O1 and above that it doesn't make a difference, with no optimizations yeah it would be more efficient to change it.